source/numeric.d.ts

```diff
@@ -113,6 +113,15 @@ Use-case: Validating and documenting parameters.
 */
 export type Negative<T extends Numeric> = T extends Zero ? never : `${T}` extends `-${string}` ? T : never;
 
+/**
```

Comment:
This should be marked as internal or properly exported publicly. Otherwise, it's hard to track what's used internally and what's public.

Reply:
It is exported and explained in readme.md